File: src/main/java/com/learnershi/rclasssocket/entity/Quiz.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

/**
 * Quiz
 *
 * @author Jae Deok
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "quiz")
public class Quiz extends ClassRoomEntity {
    private List choices;
    private Boolean isTitleInclude;
    private String questionText;
    private String quizType;

}

File: src/main/java/com/learnershi/rclasssocket/entity/Quiz.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.mongodb.core.mapping.Document

/**
 * Quiz
 *
 *
 */
@Document(collection = "quiz")
class Quiz : ClassRoomEntity() {
    private val choices: List<*>? = null
    private val isTitleInclude: Boolean? = null
    private val questionText: String? = null
    private val quizType: String? = null
}